app/policies/namespace_policy.rb

```diff
@@ -29,11 +29,14 @@ def pull?
 
   def push?
     raise Pundit::NotAuthorizedError, "must be logged in" unless user
-
+  if APP_CONFIG.enabled?("user_permission.push_images")
```

Comment:
Prefer using a guard instead (rubocop will complain about this otherwise :wink:):

```ruby
return user.admin? if APP_CONFIG.disabled?("user_permission.push_images")
```

Also, make sure to be on the same indentation level.

Reply:
done, but it was "disable" instead of "disabled". ;)